dialog/src/main/java/roboy/dialog/states/devoStates/QuestionRoboyQAState.java

```diff
@@ -103,39 +97,19 @@ public Output react(Interpretation input) {
 
     private Output reactToQuestion(Interpretation input) {
 
-        if (offeredGame) {
-            questionsAnswered++;
-            offeredGame = false;
-            if (getInference().inferSentiment(input) == Linguistics.UtteranceSentiment.POSITIVE) {
-                userWantsGame = true;
-                questionsAnswered++;
-                return Output.sayNothing().setSegue(new Segue(Segue.SegueType.PICKUP,0.5)).setEmotion(RoboyEmotion.HAPPY);
-            }
-            else {
-                return Output.sayNothing().setEmotion(RoboyEmotion.SADNESS);
-            }
-        }
-
         askingSpecifyingQuestion = false;
-        userWantsGame = false;
         questionsAnswered++;
 
         String answer = inferApiAnswer(input);
         if (!answer.isEmpty()) {
             return Output.say(answer);
         }
 
-        // catch jokes
-        if (input.getTokens().contains("joke") || input.getTokens().contains("funny")) {
-            return Output.say(PhraseCollection.JOKES.getRandomElement()).setEmotion(RoboyEmotion.positive.getRandomElement());
-        }
-
         if (input.getTokens() != null && !(input.getTokens().contains("you") || input.getTokens().contains("your"))) {
             Linguistics.ParsingOutcome parseOutcome = input.getParsingOutcome();
-
             if (parseOutcome == null) {
                 LOGGER.error("Invalid parser outcome!");
-                return Output.useFallback();
+                return Output.say("Invalid parser outcome!");
```

Comment:
don't use the hardcoded strings in states, please. they're a nightmare to track down once the system is properly deployed

Reply:
Should be fixed